tests/helpers.py

```diff
@@ -97,3 +102,78 @@ def max_pooling(_input, pool_shape, pool_stride):
                                pool_shape, k])
 
     return results
+
+class ResultsHelper:
```

Comment:
i recommend this end up in tests/benchmarks, since i think it's only a matter of time before benchmarks move out of the tests/ tree.